proposals/NNNN-rc-sdl-module.md

```diff
@@ -0,0 +1,239 @@
+         <description> This array can contain the list of RPCs that can be read/requested from this module/app. If an array is not included, it is assumed all RPCs are supported. </description>
+    </param>
+        
+</struct>
+```
+
+##### ModuleType
+```xml
+    <enum name="ModuleType">
+        <element name="CLIMATE"/>
+        <element name="RADIO"/>
+        <!-- NEW --> 
+        <element name="SDL_APP"/>
+    </enum>
+```
+
+##### ModuleData
+```xml
+    <struct name="ModuleData">
+        <description>The moduleType indicates which type of data should be changed and identifies which data object exists in this struct. For example, if the moduleType is CLIMATE then a "climateControlData" should exist</description>
+        <param name="moduleType" type="ModuleType" mandatory="true">
+        </param>
+        <!-- NEW --> 
+        <param name="moduleUUID" type="String" mandatory="false">
+            <description>This is a unique identifier for this module. 
+            </description>
+        </param> 
+        ...
+        <param name="sdlAppControlData" type="SdlAppControlData" mandatory="false">
+        </param>
+    </struct>
+
+```
+
+##### GetInteriorVehicleData
+
+```xml
+    <function name="GetInteriorVehicleData" functionID="GetInteriorVehicleDataID" messagetype="request">
+        <param name="moduleType" type="ModuleType" mandatory="true">
+            <description>
+                The type of a RC module to retrieve module data from the vehicle.
+                In the future, this should be the Identification of a module.
+            </description>
+        </param>
+       <!-- NEW --> 
+        <param name="moduleUUID" type="String" mandatory="false">
+            <description>This is a unique identifier for this module. 
+            </description>
+        </param>
+       
+       ...
+    </function>
+```
+
+
+
+
+## Potential downsides
+- App developers may have to keep track of which module to send RPCs.
+- The solution bends the guidelines of how the protocol works. 
+- We have to add some sort of identifier to modules. In this proposals they are simple UUIDs. 
+
+## Impact on existing code
+- The new type of module would have to be created and code that would allow one RPC to be combined into another packet's bulk data.
+- The proxies will need to call handle parsing the bulk data into an RPC and pass through the correct callbacks. 
+
+
+
+## Alternatives considered
+
+#### New App types
+Instead of using the already existing HMI_TYPE enum, it would be possible to introduce a new set of enums that are defined specifically for what service type is offered through its virtual RC module.
+
+#### Core Sends Bulk Data RPC
+Instead for forwarding on the SetInteriorVehicle data packet as a whole, Core would parse the packet and the bulk data. It was decode the bulk data into an RPC and form a new SDL packet that would get sent to the destination app.
```

Comment:
Please update the first sentence to "Instead of forwarding..." and update the second to "It would decode the bulk..."